pysages/colvars/core.py

```diff
@@ -157,7 +182,7 @@ def _get_nargs(function: Callable):
     return len(signature(function).parameters)
 
 
-def _build(cv: CollectiveVariable, grad=jax_grad):
+def _build(cv: CollectiveVariable, diff: bool = True):
```

Comment:
[minor point - can be ignored] diff seems like difference. can we call it `deriv`?

Reply:
Change it to `differentiate`.